client/core/types/vote.go

```diff
@@ -217,3 +218,39 @@ func (vote *Vote) String() string {
 }
 
 type Votes []*Vote
```

Comment:
Is the `VoteSet` a replacement for `Votes` or we still use both?

Reply:
We still use both. `Votes` is used where we write new data to the commit to the DB.

Reply:
Ok